server/db/models/Gig.js

```diff
@@ -36,14 +36,28 @@ module.exports = function (sequelize, DataTypes) {
 
         var positionQuery = {
           model: Gig.associations['Positions'].target,
-          attributes: ['title'],
+          attributes: ['title', 'id'],
           through: {
             attributes: ['required', 'filled'],
             as: 'statusInfo'
           }
         };
 
-        include.push(positionQuery);
+        var locationQuery = {
+          model: Gig.associations['Location'].target,
+          attributes: [
+            'id',
+            'name',
+            'addressOne',
+            'addressTwo',
+            'city',
+            'state',
+            'zipCode'
+          ],
+          as: 'Location'
+        };
+
+        include.push(positionQuery, locationQuery);
```

Comment:
http://sequelize.readthedocs.org/en/latest/api/model/   do a find for 'include' - it's partway down the page